scripts/validate-requirements.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# bash scripts/validate-requirements.sh "END_USER_EMAIL" "ORGANIZATION_ID" "BILLING_ACCOUNT_ID"
+
+# -------------------------- Variables --------------------------
+# Expected versions of the installers
+TF_VERSION="0.13.7"
+GCLOUD_SDK_VERSION="319.0.0"
+GIT_VERSION="2.25.1"
+# User Inputs
+END_USER_CREDENTIAL="$1"
+ORGANIZATION_ID="$2"
+BILLING_ACCOUNT="$3"
```

Comment:
you clous check that that these values are not null before running
and present a helper message